src/site/topic22.rst

```diff
@@ -26,6 +26,32 @@ Post-order
 Level-order
 -----------
```

Comment:
Maybe use the use _breadth_ somewhere, so it's not something totally new when they eventually see breadth first search (2nd year probably?).